libraries/wallet/wallet.cpp

```diff
@@ -758,6 +758,50 @@ class wallet_api_impl
 
       return true;
    }
+
+   signed_transaction multisig_sign_transaction( signed_transaction tx,
+                                                 bool broadcast )
+   {
+      set<public_key_type> pks = _remote_db->get_potential_signatures( tx );
+      flat_set<public_key_type> owned_keys;
+      owned_keys.reserve( pks.size() );
+      std::copy_if( pks.begin(), pks.end(),
+                    std::inserter( owned_keys, owned_keys.end() ),
+                    [this]( const public_key_type &pk ) {
+                       return _keys.find( pk ) != _keys.end();
+                    } );
+
+      set<public_key_type> approving_key_set =
+         _remote_db->get_required_signatures( tx, owned_keys );
+
+      if ( ( ( tx.ref_block_num == 0 && tx.ref_block_prefix == 0 ) ||
+             tx.expiration == fc::time_point_sec() ) &&
+           tx.signatures.empty() )
+      {
+         auto dyn_props = get_dynamic_global_properties();
+         tx.set_reference_block( dyn_props.head_block_id );
+         tx.set_expiration( fc::time_point_sec::maximum() );
```

Comment:
Need to use current (on chain) maximum value in `global_properties` but not the absolute maximum value.

Reply:
Thanks~ fixed in f61cb75

Reply:
Seems wrong.

Reply:
oh got it, trouble check it again?